pkg/controller/volume/scheduling/scheduler_binder.go

```diff
@@ -71,25 +71,25 @@ type InTreeToCSITranslator interface {
 	TranslateInTreePVToCSI(pv *v1.PersistentVolume) (*v1.PersistentVolume, error)
 }
 
-// SchedulerVolumeBinder is used by the scheduler to handle PVC/PV binding
-// and dynamic provisioning.  The binding decisions are integrated into the pod scheduling
-// workflow so that the PV NodeAffinity is also considered along with the pod's other
-// scheduling requirements.
+// SchedulerVolumeBinder is used by the scheduler VolumeBinding plugin to
+// handle PVC/PV binding and dynamic provisioning. The binding decisions are
+// integrated into the pod scheduling workflow so that the PV NodeAffinity is
+// also considered along with the pod's other scheduling requirements.
 //
-// This integrates into the existing default scheduler workflow as follows:
+// This integrates into the existing scheduler workflow as follows:
```

Comment:
`default scheduler` is misleading now, because we can configure multiple profiles for scheduler now (except the `default-scheduler` profile). all profiles share the same workflow except each one can disable/enable different plugins and args.